mypy/test/teststubtest.py

```diff
@@ -144,9 +144,38 @@ def __invert__(self: _T) -> _T: pass
 """
 
 
+@overload
 def run_stubtest(
```

Comment:
I feel like it'd be cleaner to add two separate helper functions, one that returns only the stdout and one that returns both.